playpen/dev-setup.sh

```diff
@@ -80,10 +80,14 @@ pushd pulp
 setfacl -m user:apache:rwx .
 popd; popd
 
+echo "Disabling strict mode for ssh"
+echo 'StrictModes no' | sudo tee -a /etc/ssh/sshd_config
+sudo systemctl restart sshd
```

Comment:
I do think it would be good if the script prompted the user with a y/n before doing this, since it is a security change. Something similar to the selinux prompt. Also, the prompt should not happen if StrictModes is already set to no.